tests/testsuite/fixture.go

```diff
@@ -228,6 +229,25 @@ func WipeTestingInfrastructure() {
 	deployOrWipeTestingInfrastrucure(DeleteRawManifest)
 }
 
+func waitForAllDaemonSetsReady(timeout time.Duration) {
+	checkForDaemonSetsReady := func() []string {
+		dsNotReady := make([]string, 0)
+		virtClient, err := kubecli.GetKubevirtClient()
+		util.PanicOnError(err)
+
+		dsList, err := virtClient.AppsV1().DaemonSets(k8sv1.NamespaceAll).List(context.Background(), metav1.ListOptions{})
+		util.PanicOnError(err)
+		for _, deployment := range dsList.Items {
+			if deployment.Status.DesiredNumberScheduled != deployment.Status.NumberReady {
+				dsNotReady = append(dsNotReady, deployment.Name)
+			}
+		}
+		return dsNotReady
+
+	}
+	Eventually(checkForDaemonSetsReady, timeout, 2*time.Second).Should(BeEmpty(), "There are deployments in system which are not ready.")
```

Comment:
deployments->daemonsets

Reply:
oops, fixed all the references for deployments (including some variable names)